examples/http-client-basic-authentication/http_client_basic_authentication.md

```diff
@@ -1,6 +1,6 @@
 # HTTP client - Basic authentication
 
-A client, which is secured with basic authentication can be used to connect to a secured service. The client is enriched with the `Authorization: Basic <token>` header by passing the `http:CredentialsConfig` for the `auth` configuration of the client.
+The `http:Client` can be secured with basic authentication by enriching each request from the client with the `Authorization: Basic <token>` header. The username and password for basic authentication can be specified in the `auth` field of the `http:ClientConfiguration`. Use this to communicate with the service, which is secured with basic authentication.
```

Comment:
The username and password for basic authentication can be specified in the `auth` field of the client configuration.

IMO above is better.

Reply:
+1

Reply:
Fixed via [4fdc839](https://github.com/ballerina-platform/ballerina-distribution/pull/3791/commits/4fdc83992465d9b18e44fbc5e824f2393d34dd7b)